src/types/index.ts

```diff
@@ -30,3 +34,31 @@ export interface VendorConsents {
 		};
 	};
 }
+
+export enum VendorIDs {
```

Comment:
is this a type definition? it _is_ the actual data – i wonder if it should live in `getConsentFor.ts`?

Reply:
yeah is the actual data, i moved it inside index.ts because eslint was complaining about a circular dependency. open to suggestion on how to move it back to  `getConsentFor.ts` without that error :)

Reply:
ah right, ok. how about this?

in `types/window.d.ts`, use `typeof` e.g. 

```ts
import { getConsentFor } from '../getConsentFor';
import { onConsentChange } from '../onConsentChange';
// ...
onConsentChange?: typeof onConsentChange;
getConsentFor?: typeof getConsentFor;
```

then you should be able to remove `OnConsentChange` and `GetConsentFor` types from `types/index.ts` entirely (also makes it safer, don't have to maintain the definitions).

then i think you could move the `VendorIDs` to `src/getConsentFor.ts`

Reply:
much simpler ! 💯 done